src/app/admin/adminApp.js

```diff
@@ -39,6 +55,170 @@ const authProvider = ({ signIn, signOut }) => ({
   getPermissions: () => Promise.resolve(),
 });
 
+const validateSpecialistCreation = (values) => {
+  const errors = {};
+  console.log(values);
+  if (!values.Specialization_position) {
+    errors.Specialization_position =
+      "The Specialization / position is required";
+  }
+  if (!values.full_name) {
+    errors.full_name = "The full name is required";
+  }
+  if (!values.sex) {
+    errors.sex = "The sex is required";
+  }
+  if (!values.service_format) {
+    errors.service_format = "The service format is required";
+  }
+  if (!values.full_address) {
+    errors.full_address = "The full address is required";
+  }
+  if (!values.district) {
+    errors.district = "The district is required";
+  }
+  if (!values.type_of_therapy) {
+    errors.type_of_therapy = "The type of therapy is required";
+  }
+  if (!values.free_admission) {
+    errors.free_admission = "The free admission is required";
+  }
+  if (!values.description) {
+    errors.description = "The description is required";
+  }
+  return errors;
+};
+
+export const SpecialistCreate = () => {
+  const [serviceFormat, setServiceFormat] = useState("");
+  const shouldShowAddressFields = serviceFormat !== "online";
+
+  const onServiceFormatChange = (value) => {
+    // console.log(value.target.value);
+    setServiceFormat(value.target.value);
+  };
+
+  return (
+    <Create>
+      <SimpleForm validate={validateSpecialistCreation}>
+        {/* <SelectArrayInput
+          source="Specialization_position"
+          choices={[
+            { id: "admin", name: "Психологічний консультант" },
+            { id: "u001", name: "Психотерапевт" },
+            { id: "u002", name: "Психіатр" },
+            { id: "u003", name: "Сексолог" },
+            { id: "u004", name: "Соціальний працівник" },
+          ]}
+          validate={[required()]}
+          label="Specialization / position"
+        /> */}
+        <CheckboxGroupInput
+          source="Specialization_position"
+          choices={[
```

Comment:
It would be nice to keep jsx a bit cleaner by storing all the data outside return(). For example, you can create a "specializationPositionChoices" obj to store all the options in there.